CMAKE --build issue

## snippet of Dockerfile used (using same dev env dockerfile as in rippled documentation)
RUN  git clone https://github.com/ripple/validator-keys-tool.git && cd validator-keys-tool &&  git checkout master
WORKDIR /root/validator-keys-tool
RUN conan profile new default --detect
RUN conan profile update settings.compiler.libcxx=libstdc++11 default
RUN conan remote add ripple http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod
RUN mkdir .build
WORKDIR /root/validator-keys-tool/.build
RUN conan install .. --output-folder . --build missing
RUN cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_TOOLCHAIN_FILE:FILEPATH=conan_toolchain.cmake -DCMAKE_BUILD_TYPE=Release ..

## Issue occurs here
RUN cmake --build .

## ERROR
root@c3a5ec476012:~/validator-keys-tool/.build# cmake --build .
[ 20%] Building CXX object CMakeFiles/validator-keys.dir/src/ValidatorKeys.cpp.o
In file included from /root/validator-keys-tool/src/ValidatorKeys.cpp:21:
/root/validator-keys-tool/src/ValidatorKeys.h:21:10: fatal error: xrpl/protocol/KeyType.h: No such file or directory
   21 | #include <xrpl/protocol/KeyType.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
gmake[2]: *** [CMakeFiles/validator-keys.dir/build.make:76: CMakeFiles/validator-keys.dir/src/ValidatorKeys.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:111: CMakeFiles/validator-keys.dir/all] Error 2
gmake: *** [Makefile:101: all] Error 2

## conan profile
root@c3a5ec476012:~/validator-keys-tool/.build# conan profile show default
Configuration for profile default:

[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=11
compiler.libcxx=libstdc++11
build_type=Release
[options]
[conf]
[build_requires]
[env]

building on m1 (ARM) in docker using --platform=linux/amd64 flag in dockerfile

--- a/conanfile.txt
+++ b/conanfile.txt
@@ -1,5 +1,5 @@
 [requires]
-xrpl/2.1.0
+xrpl/2.3.0-b4
 
 [generators]
 CMakeDeps

